backend/src/models/reports/rpt_PA_Fiscal_Registry.js

```diff
@@ -0,0 +1,86 @@
+const dbConnection = require("@database/databaseConnection");
+const { knex } = dbConnection();
+
+/**
+ * Gets data for the Divisional Project Reports - Project Dashboard report.
+ *
+ * @param           requestParams portfolio: Optional list of portfolio_ids to limit report to. If empty, returns data for all portfolios. date: Optional date param. to limit report to.
+ * @returns {any[]}
+ */
+
+const handleParams = (query, requestParams) => {
```

Comment:
i see where you're going with this. if there's a portfolio or another param, you simply add it here!